### PR TITLE
Redeploy Mecha

### PR DESCRIPTION
**Pull request by EmilienM**
This was previously documented, now it's automated.
This reflects what's in place today; and it was already executed
manually and documented.

Next time we have to redeploy, it'll be automated.

### CODE REVIEW

**Comment by EmilienM** [comment]
FYI @MaysaMacedo

File: configs/mecha-central.yaml [hunk 1]
```diff
@@ -22,3 +22,24 @@ standalone_extra_config:
...
... start=38.129.56.150,end=38.129.56.150 --allocation-pool start=38.129.56.94,end=38.129.56.94 --allocation-pool start=38.129.56.56,end=38.129.56.56 --allocation-pool start=38.129.56.148,end=38.129.56.148 --allocation-pool start=38.129.56.221,end=38.129.56.221 --allocation-pool start=38.129.56.193,end=38.129.56.193 --allocation-pool start=38.129.56.210,end=38.129.56.210 --allocation-pool start=38.129.56.130,end=38.129.56.130 --allocation-pool start=38.129.56.184,end=38.129.56.184 --allocation-pool start=38.129.56.67,end=38.129.56.67 --allocation-pool start=38.129.56.103,end=38.129.56.103 --allocation-pool start=38.129.56.185,end=38.129.56.185 --allocation-pool start=38.129.56.205,end=38.129.56.205 --allocation-pool start=38.129.56.84,end=38.129.56.84 --allocation-pool start=38.129.56.214,end=38.129.56.214 --allocation-pool start=38.129.56.63,end=38.129.56.63 --allocation-pool start=38.129.56.9,end=38.129.56.9 --allocation-pool start=38.129.56.202,end=38.129.56.202 --allocation-pool start=38.129.56.187,end=38.129.56.187 --allocation-pool start=38.129.56.227,end=38.129.56.227 --allocation-pool start=38.129.56.110,end=38.129.56.110 --allocation-pool start=38.129.56.145,end=38.129.56.145
```
Comment (mdbooth) [inline configs/mecha-central.yaml:45]: If you have a way to obtain this list of ips programatically, something like the following would be more maintainable:

```shell
while read ip; do
  echo --allocation-pool start="$ip",end="$ip"
done < ips.txt  | xargs openstack subnet set external-subnet --no-allocation-pool
```

Are these floating ips on a Vexxhost provider network? Can we list them?

If not, a HEREDOC which just listed them would also be more readable.

Comment (EmilienM) [reply configs/mecha-central.yaml:45]: you're right, I just found a way (in my brain) to do it programatically.

Thanks. Don't merge it now.

File: configs/mecha-central.yaml [hunk 2]
```diff
@@ -22,3 +22,24 @@ standalone_extra_config:
...
+  openstack role add --user openshift --project openshift-central member
+  openstack role add --user openshift --project openshift-az0 member
+  openstack role add --user openshift --project openshift-kuryr member
+  openstack project delete openshift
+  openstack quota set --cores 170 --fixed-ips -1 --injected-file-size -1 --injected-files -1 --instances -1 --key-pairs -1 --properties -1 --ram 790000 --gigabytes 2000 --server-groups -1 --server-group-members -1 --backups -1 --backup-gigabytes -1 --per-volume-gigabytes -1 --snapshots -1 --volumes -1 --floating-ips 20 --secgroup-rules -1 --secgroups -1 --networks -1 --subnets -1 --ports -1 --routers -1 --rbac-policies -1 --subnetpools -1 openshift-central
+  openstack quota set --cores 52 --fixed-ips -1 --injected-file-size -1 --injected-files -1 --instances -1 --key-pairs -1 --properties -1 --ram 210000 --gigabytes 2000 --server-groups -1 --server-group-members -1 --backups -1 --backup-gigabytes -1 --per-volume-gigabytes -1 --snapshots -1 --volumes -1 --floating-ips 8 --secgroup-rules -1 --secgroups -1 --networks -1 --subnets -1 --ports -1 --routers -1 --rbac-policies -1 --subnetpools -1 openshift-kuryr
```
Comment (MaysaMacedo) [inline configs/mecha-central.yaml:38]: Kuryr tenant is configured with 250000 of ram and 60 of cores. I will update the doc to reflect that

File: configs/mecha-central.yaml [hunk 3]
```diff
@@ -22,3 +22,24 @@ standalone_extra_config:
...
+  openstack image show centos8-stream || wget https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-GenericCloud-8-20210210.0.x86_64.qcow2 && openstack image create --public --disk-format qcow2 --file CentOS-Stream-GenericCloud-8-20210210.0.x86_64.qcow2 centos8-stream && rm -f CentOS-Stream-GenericCloud-8-20210210.0.x86_64.qcow2
+  openstack project create openshift-central
+  openstack project create openshift-az0
+  openstack project create openshift-kuryr
+  openstack role add --user openshift --project openshift-central member
+  openstack role add --user openshift --project openshift-az0 member
+  openstack role add --user openshift --project openshift-kuryr member
+  openstack project delete openshift
+  openstack quota set --cores 170 --fixed-ips -1 --injected-file-size -1 --injected-files -1 --instances -1 --key-pairs -1 --properties -1 --ram 790000 --gigabytes 2000 --server-groups -1 --server-group-members -1 --backups -1 --backup-gigabytes -1 --per-volume-gigabytes -1 --snapshots -1 --volumes -1 --floating-ips 20 --secgroup-rules -1 --secgroups -1 --networks -1 --subnets -1 --ports -1 --routers -1 --rbac-policies -1 --subnetpools -1 openshift-central
```
Comment (MaysaMacedo) [inline configs/mecha-central.yaml:37]: central tenant has 150 of cores

**Review by MaysaMacedo — COMMENTED** [review]
Minor quotas update needed:

File: configs/mecha-central.yaml (hunk 2 above)
Comment (MaysaMacedo) [reply configs/mecha-central.yaml:38]: seems the quota section is not there, but I double checked the values in the cloud

Comment (EmilienM) [reply configs/mecha-central.yaml:38]: ok thanks

File: configs/mecha-central.yaml [hunk 4]
```diff
@@ -22,3 +22,23 @@ standalone_extra_config:
...
+  openstack image show centos8-stream || wget https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-GenericCloud-8-20210210.0.x86_64.qcow2 && openstack image create --public --disk-format qcow2 --file CentOS-Stream-GenericCloud-8-20210210.0.x86_64.qcow2 centos8-stream && rm -f CentOS-Stream-GenericCloud-8-20210210.0.x86_64.qcow2
+  openstack project create openshift-central
+  openstack project create openshift-az0
+  openstack project create openshift-kuryr
+  openstack role add --user openshift --project openshift-central member
+  openstack role add --user openshift --project openshift-az0 member
+  openstack role add --user openshift --project openshift-kuryr member
+  openstack project delete openshift
+  openstack quota set --cores 150 --fixed-ips -1 --injected-file-size -1 --injected-files -1 --instances -1 --key-pairs -1 --properties -1 --ram 790000 --gigabytes 2000 --server-groups -1 --server-group-members -1 --backups -1 --backup-gigabytes -1 --per-volume-gigabytes -1 --snapshots -1 --volumes -1 --floating-ips 20 --secgroup-rules -1 --secgroups -1 --networks -1 --subnets -1 --ports -1 --routers -1 --rbac-policies -1 --subnetpools -1 openshift-central
```
Comment (EmilienM) [inline configs/mecha-central.yaml:36]: need to update ram to be `395000` since we removed the compute node on az1.

Comment (EmilienM) [reply configs/mecha-central.yaml:36]: need to update CPU to be `75` as well.

Comment (EmilienM) [reply configs/mecha-central.yaml:36]: and gigabytes to `1000`

**Pull request merged**